index.html

```diff
@@ -1,13 +1,81 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-  <link rel="stylesheet" href="styles/index.css">
-</head>
-<body>
-  <h1>Quiz-App</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App start page</title>
+    <link rel="stylesheet" href="styles/index.css" />
+  </head>
+  <body class="app-grid">
+    <header id="#starthome" class="top">
+      <h1 class="top quiz-app-heading">Quiz-App</h1>
+    </header>
+
+    <main class="app-grid__scrollarea h-100vh">
+      <section class="quiz-app-card">
+        <article class="quiz-app-card-question">
+          <h2 class="quiz-app-card__heading">Question:</h2>
+          <p class="quiz-app-card-question--text"></p>
+        </article>
+        <figure class="bookmark-container">
+          <img src="" alt="" width="" height="" class="bookmark--on" />
+        </figure>
+        <button class="quiz-app-card-answer--show"></button>
+        <button class="quiz-app-card-answer--hide"></button>
+        <p class="quiz-app-card-question--answer"></p>
+        <div class="quiz-app-card-advise-tag__container">
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+          <button class="quiz-app-card-advise-tag" type="text"></button>
+        </div>
+      </section>
+    </main>
+
+    <nav class="nav-container">
+      <ul class="nav-container__list">
+        <li class="nav-container__listitem">
```

Comment:
nice :) here you used BEM quite good already...I just would recommend to change `<li class="nav-container__listitem>` to `<li class="nav-container__item"> `

Reply:
this would be shorter but i wanted to introduce the parent child connection and an easy declaration to find it again in storybook